src/interactions/pointerInteraction.ts

```diff
@@ -34,19 +35,32 @@ export module Interactions {
       this._touchDispatcher = null;
     }
 
-    private _handlePointerEvent(p: Point) {
+    private _handleMouseEvent(p: Point, e: MouseEvent) {
+      let insideSVG = this._mouseDispatcher.eventInsideSVG(e);
+      this._handlePointerEvent(p, insideSVG);
+    }
+
+    private _handleTouchEvent(p: Point, e: TouchEvent) {
+      let insideSVG = this._touchDispatcher.eventInsideSVG(e);
+      this._handlePointerEvent(p, insideSVG);
+    }
```

Comment:
the only difference is that it checks against `mouseDispatcher` for `MouseEvent` and `touchDispatcher` for `TouchEvent`.

I could move this check inside `_handlePointerEvent` but I feel it's hacky to check the type of `e` and decide which `Dispatcher` to use